bootstrap_cfn/fab_tasks.py

```diff
@@ -312,3 +314,45 @@ def rsync():
             'cloudformation.sls'),
         local_path=cf_sls,
         use_sudo=True)
+
+
+@task
+def update_certs(force=False, verbose=False):
+    """
+    Update the ssl certificates with those in the config file.
+    Also handle settings the certificates on ELB's
+    """
+    if verbose:
+        logging.basicConfig(level=logging.INFO)
```

Comment:
Rather than setting a flag called verbose here, it would be better to just use the logger with different levels. Like `logging.info` when you want it to appear for everyone and `logging.debug` when you want it only if the verbosity is increased. Then we can set the level globally in the fabfile maybe?